tailcfg/tailcfg.go

```diff
@@ -1221,11 +1222,22 @@ type CapGrant struct {
 	CapMap PeerCapMap `json:",omitempty"`
 }
 
-// PeerCapability is a capability granted to a node by a FilterRule.
-// It's a string, but its meaning is application-defined.
-// It must be a URL, like "https://tailscale.com/cap/file-sharing-target" or
-// "https://example.com/cap/read-access".
-type PeerCapability string
+// CapabilityName is a string used to identify a specific capability. Its
+// meaning is application-defined.
+//
+// It must be a URL like "https://tailscale.com/cap/file-send", or a well-known
+// capability name like 'funnel'.
+type CapabilityName string
+
+// NodeCapability is a capability granted to a node.
+//
+// Unlike PeerCapability, NodeCapability is not in context of a peer and is
+// granted to the node itself.
+type NodeCapability CapabilityName
```

Comment:
trying to avoid "node attribute"? don't want to use that name or at least mention it as another name for this?